scripts/_redirects

```diff
@@ -164,7 +164,7 @@
 /docs/build-storage    https://docs.polkadot.com/develop/    301
 /docs/build-dapp    https://docs.polkadot.com/develop/#application-developers    301
 /docs/build-pdk    https://docs.polkadot.com/develop/parachains/   301
-/docs/build-light-clients    /build/build-light-clients    301
+/docs/build-light-clients    /general/web3-and-polkadot/    301
```

Comment:
FYI soon we should have light client page on the docs: https://github.com/polkadot-developers/polkadot-docs/pull/585